src/mat2tsk.py

```diff
@@ -199,8 +199,7 @@ def date_samples(tsk_in, tsk_out):
             node_time[u] = max(node_time[v] + 1 for v in tree.children(u))
 
     tables = ts.dump_tables()
-    tables.metadata = {"sc2ts": {"date": str(time_zero)}}
-    print("Setting time zero = ", str(time_zero))
```

Comment:
Can you leave the print in please, this is helpful in debugging the pipeline

Reply:
Ah, sorry, I didn't know that had snuck in.